postfix/files/tls_policy

```diff
@@ -0,0 +1,17 @@
+#
+# This file is managed by salt
+#
+
+{% set config = salt['pillar.get']('postfix:tls_policy', {}) -%}
+{% macro set_parameter(parameter, default=None) -%}
+{% set value = config.get(parameter, default) -%}
```

Comment:
I'd strongly recommend avoiding the use of `config.get`, since that can get easily get confused with:

* https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.config.html#salt.modules.config.get

I'll still merge anyway but it would be helpful if you can change this in a subsequent PR.

Reply:
@myii Will do, thianks for the advice!